pkg/ipmi/ocp/set_vpd.go

```diff
@@ -10,15 +10,25 @@ import (
 	"log"
 	"os"
 	"os/exec"
+	"os/signal"
+	"syscall"
 )
 
+func handler(c <-chan os.Signal) {
+	<-c
+	log.Printf("ignoring SIGINT during flash write to prevent corruption")
+}
```

Comment:
This will only ever catch one single signal, and if no signal comes the goroutine will leak and sit there waiting forever.
```suggestion
func handler(c <-chan os.Signal) {
	for range c {
		log.Printf("ignoring SIGINT during flash write to prevent corruption")
	}
}
```

Reply:
Done.